From 9cdc8eed9c5ec68be7e03a77f80209ed4f9915da Mon Sep 17 00:00:00 2001
From: Dmitry Dementyev <dementyev@google.com>
Date: Mon, 6 Mar 2023 22:01:20 +0000
Subject: [PATCH] [automerge] Convert argument to Intent in car settings
 account removal. 2p: a7a4b16b8f

Original change: https://googleplex-android-review.googlesource.com/c/platform/packages/apps/Car/Settings/+/21804481

Bug: 265798353
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:d1d98e8ab60f8152751f7f49301a8cfd14f89c77)
Merged-In: I6c42b64cc837843a1a45d6bcd8ada481cc34272e
Change-Id: I6c42b64cc837843a1a45d6bcd8ada481cc34272e
---
 .../settings/accounts/AccountDetailsPreferenceController.java   | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/car/settings/accounts/AccountDetailsPreferenceController.java b/src/com/android/car/settings/accounts/AccountDetailsPreferenceController.java
index 1685b7d66..f07c94712 100644
--- a/src/com/android/car/settings/accounts/AccountDetailsPreferenceController.java
+++ b/src/com/android/car/settings/accounts/AccountDetailsPreferenceController.java
@@ -68,7 +68,7 @@ public class AccountDetailsPreferenceController extends AccountDetailsBasePrefer
                     Intent removeIntent = result.getParcelable(AccountManager.KEY_INTENT);
                     if (removeIntent != null) {
                         done = false;
-                        getFragmentController().startActivityForResult(removeIntent,
+                        getFragmentController().startActivityForResult(new Intent(removeIntent),
                                 REMOVE_ACCOUNT_REQUEST, this);
                     } else {
                         success = future.getResult().getBoolean(AccountManager.KEY_BOOLEAN_RESULT);
-- 
2.41.0.rc0.172.g3f132b7071-goog

